shared/components/TicketList/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from 'react';
+import { formatDate } from '../../utilities/date';
+import classnames from 'classnames';
+
+export const TicketPrice = (ticket) => <strong>{ticket.available ? `£${ticket.price}` : 'SOLD OUT'}</strong>
+
+export function BuyTickets({ tickets }) {
```

Comment:
Prob should use the arrow function syntax as we have used that for all other components.